nameable_decorator/decorators.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'base_decorator'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    super()
+    @nameable.correct_name.capitalize
+  end
+end
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    super()
+    @nameable.correct_name[0, 9]
+  end
```

Comment:
- [ ] Kindly ensure that this method trims the number of characters if it is greater than 10 `(>10)` else it returns `@namable.correct_name`. 👍